apps/calendar/test/marionette/lib/views/week.js

```diff
@@ -77,6 +83,153 @@ Week.prototype = {
     });
   },
 
+  /**
+   * This returns a boolean that is true if the hour square is fully visible.
+   *
+   * @param {Number} hour The hour to locate
+   * @returns {Boolean} Whether the square is fully visible.
+   */
+  _isHourDisplayed: function(hour) {
```

Comment:
IMO, tests shouldn't require documentation. The main reasons are:
* These tests are not part of a public API. The people who will read them, will only do it though the raw code, not an external website.
* The comments often can be replaced by more explicit names. This will prevent scrolling around the code to understand what the test is supposed to do.

Here, I think we can name this function: `_isHourSquareFullyDisplayed`. What do you think?